### ISSUE TITLE
Cannot Load Big SpriteFonts

### ISSUE THREAD

**Issue by GnicoJP** [MonoGame/MonoGame#5544]
Hi,
  
I am developing a game it can draw japanese texts, so I usually make spritefonts which use japanese charactor drawing. Size of files for the fonts are too large (more than 120 Megabytes) to load.
With previous versions of MonoGame, the fonts could be loaded using DirectX, but they could not be loaded in MonoGame 3.6 (current version).  
Then I tried OpenGL + MonoGame 3.6 conbination, it was suceeded.
I would like to know why, and hope this BUG to be fixed. 
  
Following is the thrown exception, and [the project which is in the trouble](https://drive.google.com/file/d/0B8cVJK3HJSj4X3ZmU2JtUWItWlU/view?usp=sharing) is attatched.   

![Exception_Image](https://cloud.githubusercontent.com/assets/26196129/23585159/c57dac4e-01b9-11e7-8375-814a2cd4c608.jpg)

**SharpDX.SharpDXException**
HRESULT: 0x80070057
ApiCode: E_INVALIDARG/Invalid Arguments
Messsage: Invalid Arguments
StackTrace:  
>   At SharpDX.Result.CheckError()  
   At SharpDX.Direct3D11.Device.CreateTexture2D(Texture2DDescription& descRef, DataBox[] initialDataRef, Texture2D texture2DOut)  
   At Microsoft.Xna.Framework.Graphics.Texture2D.CreateTexture()  
   At Microsoft.Xna.Framework.Graphics.Texture2D.PlatformSetData\[T](Int32 level, Int32 arraySlice, Rectangle rect, T[] data, Int32 startIndex, Int32 elementCount)  
   At Microsoft.Xna.Framework.Content.Texture2DReader.Read(ContentReader reader, Texture2D existingInstance)  
   At Microsoft.Xna.Framework.Content.ContentTypeReader\`1.Read(ContentReader input, Object existingInstance)  
   At Microsoft.Xna.Framework.Content.ContentReader.InnerReadObject\[T](T existingInstance)  
   At Microsoft.Xna.Framework.Content.SpriteFontReader.Read(ContentReader input, SpriteFont existingInstance)  
   At Microsoft.Xna.Framework.Content.ContentTypeReader\`1.Read(ContentReader input, Object existingInstance)  
   At Microsoft.Xna.Framework.Content.ContentReader.InnerReadObject\[T](T existingInstance)  
   At Microsoft.Xna.Framework.Content.ContentReader.ReadObject\[T]()  
   At Microsoft.Xna.Framework.Content.ContentReader.ReadAsset\[T]()  
   At Microsoft.Xna.Framework.Content.ContentManager.ReadAsset\[T](String assetName, Action`1 recordDisposableObject)  
   At Microsoft.Xna.Framework.Content.ContentManager.Load\[T](String assetName)  
   At MyProject.Game.LoadContent() : Line 54  
   At Microsoft.Xna.Framework.Game.Initialize()  
   At MyProject.Game.Initialize() : Line 38  
   At Microsoft.Xna.Framework.Game.DoInitialize()  
   At Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior)  
   At Microsoft.Xna.Framework.Game.Run()  
   At MyProject.Program.Main() : Line 17

**Comment by nkast** [comment]
Do you set the GraphicsProfile in any way?
What size is the Font.Texture.Width/Height when you load it in OpenGL (or in 3.5)?

Please try the dll from #5451 to see if you get a different error.
http://teamcity.monogame.net/repository/download/MonoGame_PackagingWindows/37874:id/MonoGameSetup.exe

**Comment by GnicoJP** [comment]
Thank you for your quick reply.
GraphicsProfile was Reach. (I didn't set intentionally.)
Then I switched to HiDef, it can load using DirectX !
My problem was solved!
Thank you.

P.S.
The Font.Texture.Width/Height are 8192 * 4096 in OpenGL + MonoGame 3.6 .

I tried the dll from #5451, it throws a better exception. 👍 
![Img](https://cloud.githubusercontent.com/assets/26196129/23585502/354e198c-01c4-11e7-8674-7db3e372e0de.jpg)
